src/core_ocean/mode_forward/mpas_ocn_time_integration_rk4.F

```diff
@@ -277,31 +279,37 @@ subroutine ocn_time_integrator_rk4(domain, dt)!{{{
                call mpas_pool_get_array(tracersPool, trim(groupItr % memberName), tracersNew, 2)
 
                if ( associated(tracersCur) .and. associated(tracersNew) ) then
+                  !$omp parallel
                   !$omp do schedule(runtime) private(k)
                   do iCell = 1, nCells  ! couple tracers to thickness
                      do k = 1, maxLevelCell(iCell)
                         tracersNew(:, k, iCell) = tracersCur(:, k, iCell) * layerThicknessCur(k, iCell)
                      end do
                   end do
                   !$omp end do
+                  !$omp end parallel
                end if
             end if
          end do
 
          if (associated(highFreqThicknessCur)) then
+              !$omp parallel
               !$omp do schedule(runtime)
               do iCell = 1, nCells
                  highFreqThicknessNew(:, iCell) = highFreqThicknessCur(:, iCell)
               end do
               !$omp end do
+              !$omp end parallel
          end if
 
          if (associated(lowFreqDivergenceCur)) then
-              !$omp do schedule(runtime)
+              !$omp parallel
+              !$omp do schedule(runtime) private(iCell)
```

Comment:
@mattdturner when I looked through this PR, it appeared that you were intentionally adding the main index loop to the private list, so I thought that went with the addition of the parallel line before it. Are you saying that we don't need to put iCell on the private list here, or in all the other places in this PR?

Reply:
Correct, the `iCell` does not need to be in the private list here.  I thought that I had removed all of the main loop indices from private lists, but it looks like there are quite a few that I missed.

Reply:
Sorry for the extra work, but could you remove all those? Otherwise it leads to confusion. I think you could use sed within a shell script to automate it on all the files.

Reply:
Sure.  I'll take care of that today and push the changes once I'm done.

Reply:
I just pushed changes that remove the main loop index from the `!$omp private` directives.  It PASSED the thread test when compiled with DEBUG using Intel v19 on Cori